packages/ember-simple-auth/tests/unit/internal-session-test.js

```diff
@@ -23,307 +22,343 @@ describe('InternalSession', () => {
+          assert.deepEqual(session.get('authenticated'), { some: 'property', authenticator: 'authenticator:test' });
         });
 
-        it('persists its content in the store', async function() {
+        test('persists its content in the store', async function(assert) {
           await store.persist({ authenticated: { authenticator: 'authenticator:test' }, someOther: 'property' });
           await session.restore();
           let properties = await store.restore();
 
           delete properties.authenticator;
 
-          expect(properties).to.eql({ authenticated: { some: 'property', authenticator: 'authenticator:test' }, someOther: 'property' });
+          assert.deepEqual(properties, { authenticated: { some: 'property', authenticator: 'authenticator:test' }, someOther: 'property' });
         });
 
-        it('persists the authenticator factory in the store', async function() {
+        test('persists the authenticator factory in the store', async function(assert) {
           await session.restore();
           let properties = await store.restore();
 
-          expect(properties.authenticated.authenticator).to.eql('authenticator:test');
+          assert.equal(properties.authenticated.authenticator, 'authenticator:test');
         });
 
-        it('does not trigger the "authenticationSucceeded" event', async function() {
+        test('does not trigger the "authenticationSucceeded" event', async function(assert) {
           let triggered = false;
           session.one('authenticationSucceeded', () => (triggered = true));
           await session.restore();
 
-          expect(triggered).to.be.false;
+          assert.notOk(triggered);
         });
 
-        itHandlesAuthenticatorEvents(async() => {
+        itHandlesAuthenticatorEvents(async function() {
           await session.restore();
         });
       });
 
-      describe('when the authenticator rejects restoration', function() {
-        beforeEach(function() {
+      module('when the authenticator rejects restoration', function(hooks) {
+        hooks.beforeEach(function() {
           sinon.stub(authenticator, 'restore').returns(RSVP.reject());
         });
 
         itDoesNotRestore();
       });
     });
 
-    describe('when the restored data does not contain an authenticator factory', function() {
+    module('when the restored data does not contain an authenticator factory', function() {
       itDoesNotRestore();
     });
 
-    describe('when the store rejects restoration', function() {
-      beforeEach(function() {
+    module('when the store rejects restoration', function(hooks) {
+      hooks.beforeEach(function() {
         sinon.stub(store, 'restore').returns(RSVP.Promise.reject());
       });
 
-      it('is not authenticated', async function() {
-        await session.restore();
+      test('is not authenticated', async function(assert) {
+        try {
+          await session.restore();
+        } catch (e) {
+          // ignore
+        }
```

Comment:
Can we remove the `try`/`catch`? I assume we want this to raise if `restore` raises an error?